module.txt

```diff
@@ -77,6 +77,10 @@
             "id": "Gooey",
             "minVersion": "1.0.0"
         },
+        {
+            "id": "GrowingFlora",
```

Comment:
since growing flora has a hard dependency on ClimateConditions this will automatically enable the player effects such as slow down due to heat or cold - do we want that right now in Metal Renegades?